docs/contributing-to-documentation.md

```diff
@@ -47,8 +47,15 @@ ZIO contributors are encouraged to use this approach for introducing new documen
 1. First, we need to clone the ZIO project on our machine:
 
 ```bash
-git clone git@github.com:zio/zio.git
-cd zio
+$ git clone git@github.com:zio/zio.git
+$ cd zio
+```
+
+if the above throws an error , try cloning via https:
```

Comment:
I prefer using only one section for cloning the GitHub repo. We can use the https scheme and omit the alternative section.